lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -438,6 +456,109 @@ def restart_services(self, services):
         """
         self._magmad_client.restart_services(services)
 
+    def create_mme_config_backup(self):
+        """
+        This function creates a backup of default MME configuration file,
+        which can later be used to restore the original configuration
+        """
+
+        print("Creating a backup of MME configuration file")
+        self.exec_command(
+            "sudo cp -n "
+            + self.mme_config_file
+            + " "
+            + self.mme_config_file
+            + "_bak"
+        )
+
+    def reset_mme_config_file(self):
+        """
+        Reset the MME configuration from the backup configuration file
+        """
+
+        print("Restoring MME configuration from backup configuration file")
+        self.exec_command(
+            "sudo cp " + self.mme_config_file + "_bak " + self.mme_config_file
+        )
+
+    def clear_default_plmn_tac_configuration(self):
+        """
+        Remove default PLMN and TAC from MME configuration file
+        """
+
+        print("Removing default GUMMEI_LIST from MME configuration")
+        self.exec_command(
+            "sudo sed -i '/GUMMEI_LIST/{n;d}' " + self.mme_config_file
```

Comment:
No need for sudo here

Reply:
Removed sudo from the command and updated the PR